extlinks/aggregates/management/helpers/aggregate_archive_command.py

```diff
@@ -0,0 +1,307 @@
+import datetime
```

Comment:
Question: Should this have the `--object-storage-only` flag like the archive link events command? This might be out of scope, and the PR is already large, but a follow-up PR would be nice.

Reply:
I decided to add it to this patch since the changes to add this option weren't very big.